### PR TITLE
Release notes for v0.1.12

### PR DESCRIPTION
**Pull request by Gedochao**

### CODE REVIEW

**Review by lwronski — COMMENTED** [review]
There is missing section about changing of behaviors `version` command. We should mention, that there exists `cliVersion` and `scalaVersion` flags to print machine-readable output.

File: website/docs/release_notes.md [hunk 1]
```diff
@@ -5,6 +5,91 @@ sidebar_position: 99
...
+```
+Added in [#1227](https://github.com/VirtusLab/scala-cli/pull/1227) by [wleczny](https://github.com/wleczny)
+
+### Enable turning `--interactive` mode on permanently
+It is now possible to set the `--interactive` mode on by default, so that passing it explicitly isn't necessary.
+
+The next time when you run a command with the `--interactive` option set to on, Scala CLI will suggest to turn it on
+permanently. 
+
+This is recommended for environments where `scala-cli` is used by a human user only. (and not by any automation)
```
Comment (BlackAnubis7) [inline website/docs/release_notes.md]:
```suggestion
This is recommended for environments where `scala-cli` is used by a human user only (and not by any automation).
```
[NIT] I think it looks more natural when not separated from the rest of the sentence

File: website/docs/release_notes.md [hunk 2]
```diff
@@ -5,6 +5,91 @@ sidebar_position: 99
...
+
+```text
+$ scala-cli-prototype . --interactive
```
Comment (alexarchambault) [inline website/docs/release_notes.md]: ?
```suggestion
$ scala-cli . --interactive
```

File: website/docs/release_notes.md (hunk 1 above)
Comment (Gedochao) [reply website/docs/release_notes.md]: fixed

File: website/docs/release_notes.md (hunk 2 above)
Comment (Gedochao) [reply website/docs/release_notes.md]: ugh, bad copy-paste.
fixed

File: website/docs/release_notes.md [hunk 3]
```diff
@@ -5,6 +5,134 @@ sidebar_position: 99
...
+```text
+$ scala-cli --version
+Scala CLI version 0.1.11
+Default Scala version: 3.1.3
+$ scala-cli -version
+Scala CLI version 0.1.11
+Default Scala version: 3.1.3
+$ scala-cli version
+Scala CLI version 0.1.11
+Default Scala version: 3.1.3
```
Comment (lwronski) [inline website/docs/release_notes.md]:
```suggestion
$ scala-cli --version
Scala CLI version 0.1.12
Default Scala version: 3.1.3
$ scala-cli -version
Scala CLI version 0.1.12
Default Scala version: 3.1.3
$ scala-cli version
Scala CLI version 0.1.12
Default Scala version: 3.1.3
```

**Review by lwronski — APPROVED** [review]

File: website/docs/release_notes.md (hunk 3 above)
Comment (Gedochao) [reply website/docs/release_notes.md]: fixed

**Review by BlackAnubis7 — APPROVED** [review]

**Pull request merged**